src/arch/xtensa/Kconfig

```diff
@@ -17,6 +17,12 @@ config SMP
 	help
 	  Indicates that architecture uses multiple cores
 
+config HAVE_SLAVE_CORE_ROM
+	bool "Have slave core rom"
```

Comment:
It's platform capability, not something that should be costumized by user. It should be hidden option and selected by platforms in platform/Kconfig.

Reply:
@jajanusz done, please check